autogrow4-4.0.2/autogrow/operators/operations.py

```diff
@@ -992,26 +881,6 @@ def make_pass_through_list(vars, smiles_from_previous_gen_list,
             num_elite_to_advance_from_previous_gen,
         )
 
-    elif generation_num != 0 and has_dock_score is False:
-        # Take the 1st num_elite_to_advance_from_previous_gen number of
-        # molecules from ligands_which_passed_filters
-        random.shuffle(ligands_which_passed_filters)
-        list_of_ligands_to_advance = []
-        for x in range(0, num_elite_to_advance_from_previous_gen):
-            selected_mol = ligands_which_passed_filters[x]
-            list_of_ligands_to_advance.append(selected_mol)
-
-    elif generation_num != 0 and has_dock_score is True:
-        # Use the make_seed_list function to select the list to advance. This
-        # list will be chosen strictly by
-        list_of_ligands_to_advance = make_seed_list(
-            vars,
-            ligands_which_passed_filters,
-            generation_num,
-            0,
-            num_elite_to_advance_from_previous_gen,
-        )
-
     if generation_num == 0:
```

Comment:
This if doesn't do anything, it just returns list_of_ligands_to_advance no matter what, entire thing can just be replaced with `return list_of_ligands_to_advance`